web_external/views/map/MapPanel.js

```diff
@@ -336,6 +336,7 @@ const MapPanel = Panel.extend({
                             layer.removeAllAnnotations();
                         }
                     });
+                    prompt.find('.bootbox-body input').attr('maxlength', '30');
```

Comment:
Out of curiosity, why 30?  I know wiredTiger supports up to 1023 bytes "including structural overhead"; probably half that would be pretty safe.

Reply:
I might be wrong. I was thinking other characters from other language takes more bytes than English letters, so I choose it to make it safe. But if 30 is too limiting, I think we change it to a bigger number. (A failed creating boundary is not that devastating)

Reply:
Sure -- I think wiredTiger uses UTF-8, which would mean that *in theory* a codepoint could take 6 bytes, but in practice 4 is sufficient.  How about we make it 250?  It is large enough most people shouldn't have an issue and small enough Mongo shouldn't have an issue.  I only discovered this because I intentionally paste absurd data into forms.

Reply:
Sounds good. New commit pushed.